mantidproject/mantidimaging
# Previews in filter window

Fixes #120

To test:
- Load some data (preferably multiple, obviously different datasets)
- Open the filters UI
- Play with data selection and preview controls, try to make it break
- Play with some filters (circular mask, intensity cutoff and rebin would give obvious results)
- Apply some filters, make sure the stack in the main window updates
- Load and close stacks whilst the filter UI is open

Known issues (these will be fixed, however I'd like to have something for the scientists to use in the meantime): #168
